jwk.go

```diff
@@ -145,3 +166,56 @@ func (key *rawJsonWebKey) fromEcPublicKey(pub *ecdsa.PublicKey) error {
 
 	return nil
 }
+
+func (jwk rawJsonWebKey) rsaPrivateKey() (*rsa.PrivateKey, error) {
+	if jwk.D == nil || jwk.P == nil || jwk.Q == nil {
+		return nil, fmt.Errorf("square/go-jose: invalid RSA private key, missing values")
+	}
+
+	rv := new(rsa.PrivateKey)
+	pubkey, err := jwk.rsaPublicKey()
+	if err != nil {
+		return nil, err
+	}
+	rv.PublicKey = *pubkey
+	rv.D = jwk.D.bigInt()
+	rv.Primes = make([]*big.Int, 2)
+	rv.Primes[0] = jwk.P.bigInt()
+	rv.Primes[1] = jwk.Q.bigInt()
```

Comment:
Would be preferable to build a struct here directly, and avoid calling `jwk.rsaPublicKey()`, e.g like:

```Go
priv := &rsa.PrivateKey{
  PublicKey: rsa.PublicKey{
    N: jwk.N.bigInt(),
    E: jwk.E.toInt(),
  },
  D: jwk.D.bigInt(),
  Primes: []*big.Int{
    jwk.P.bigInt(),
    jwk.Q.bigInt(),
  },
}
```

Also, this should set the `Precomputed` field (type `rsa.PrecomputedValues`) if DP / DQ / QI values are present.